src/rundown.ts

```diff
@@ -246,6 +246,12 @@ export interface IBlueprintPieceGeneric {
 	toBeQueued?: boolean
 	/** Array of items expected to be played out. This is used by playout-devices to preload stuff. */
 	expectedPlayoutItems?: ExpectedPlayoutItemGeneric[]
+	/** When queued, should the adlib autonext */
+	adlibAutoNext?: boolean
+	/** When queued, how much overlap with the next part */
+	adlibAutoNextOverlap?: number
+	/** When queued, block transition at the end of the part */
+	adlibDisableOutTransition?: boolean
```

Comment:
The "disable out transition" feature seems to be very specific, does it make sense to implement it, only to port it to AdLib Actions/Functions later, which I think is the more universal solution?

Reply:
I think these should be added, as it kind of serves a different use case to the adlib-actions (for now at least). these properties will be used when using the sticky hotkeys, which could be relevant for tv2 if they want to recall the last transition